tests/unit/extend/find-many-test.js

```diff
@@ -0,0 +1,50 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import ApplicationAdapter from "../../helpers/properties/application-adapter";
+import { create } from 'ember-cli-page-object';
+import { findMany } from 'ember-cli-page-object/extend';
+import { findAll } from '@ember/test-helpers';
```

Comment:
> Could not find module `@ember/test-helpers` imported from `dummy/tests/unit/extend/find-many-test`

fails for configuration w/o `@ember/test-helpers` installed. you can follow this pattern: https://github.com/san650/ember-cli-page-object/blob/629ddd3787b875567a011ad490805d348e9c2811/tests/integration/deprecations/comma-separated-selector-test.js#L7-L8

to only run the tests when `@ember/test-helpers` present